_posts/2016-09-14-supercharged-seo-with-drupal-8.md

```diff
@@ -0,0 +1,128 @@
+
+- HTML5 themes with new semantic elements in core templates
+
+- Support for the new form elements to Drupal's Form API
+
+- Rich media handling with \<video> and \<audio> elements
+
+- ARIA roles in markup to improve accessibility
+
+- Resource Description Framework (RDF) support that provides a standardized model for data interchange and facilitates [Schema.org](http://schema.org/) mappings
+
+- Twig theming engine - makes it harder for developers to create messy, non-semantic code
+
+#### Content-as-a-Service
+
+Another exciting new feature of Drupal 8 is a [flexible content delivery](https://drupal.com/feature/content-as-a-service).
+
+Today, content owners want to get their content to as many platforms and channels as possible: web, mobile, social networks, smart devices, etc. It is expensive to have a separate solution for every channel. It is much more efficient to have a single editorial team and single software platform that allows for well-organized content management. Drupal 8 and its content-as-a-service capability provides a one-stop solution where content is created and managed via unified web-interface and then consumed by other channels with minimal effort.
+
+#### Drupal 8  Multilingual Capabilities
+
+To reach audiences from around the world, companies need to speak to users in their native language. In 2017, producing content in English language [is not enough](http://www.bloggingpro.com/archives/2016/01/04/why-having-a-multilingual-website-is-the-smartest-marketing-decision/), even if English is considered an internationally accepted language. The United States is now the world's second largest Spanish-speaking country after Mexico, which amplifies the necessity of serving multilingual content for U.S. based audience. To help put things in perspective we checked recent statistics.
+
+> [English is a #1 language used in the Web, but it only amounts to 26.3% of the online market.](http://www.internetworldstats.com/stats7.htm)
+
+> [There are 41 million native Spanish speakers in the U.S. Around 79% of them using search engines on a daily basis for gathering information about a future purchase.](https://www.thinkwithgoogle.com/articles/new-research-shows-how-to-connect-with-digital-hispanics-online.html)
+
+Reaching global audience with Drupal has never been this easy! Previous versions of Drupal had partial support for multilingual websites. Luckily, Drupal 8 had a fundamental overhaul of its [multilingual system](http://www.drupal8multilingual.org/features). Every single component is translatable out of the box in Drupal core without any additional modules. Drupal core natively supports 94 languages. Also, the administration interface is now entirely translatable. Media assets (files or images), can now be assigned to a language or shared between languages. This gives a huge advantage to businesses that aim to reach global audience.
```

Comment:
Should be "Reaching _a_ global audience...", and "...aim to reach _a_ global audience."